templates/notifications/front.html.twig

```diff
@@ -42,11 +42,24 @@
                 </button>
             </div>
         </div>
-
     </div>
 
     {% for notification in notifications %}
-        <div class="{{ html_classes('section section--small notification', {'opacity-50': notification.status is not same as 'new' }) }}">
+        <div class="{{ html_classes('section section--small notification', {'opacity-50': notification.status is not same as 'new' }) }}"
+            {% if not notification.entry.magazine.apId and not notification.post.magazine.apId and (app.user.admin or app.user.moderator) %}
```

Comment:
did you also tested on if the variable is not-set / null or empty?? And if these checks are not causing any 503 errors?

Reply:
Yes I tested the lot and it's live on my instance, the variables are always set, which surprised me. Check me in mbin development as there is one weird situation with comments

Reply:
> Yes I tested the lot and it's live on my instance, the variables are always set, which surprised me. 

Well, because you are an admin? Try to login with a regular account?

Reply:
> > Yes I tested the lot and it's live on my instance, the variables are always set, which surprised me.
> 
> Well, because you are an admin? Try to login with a regular account?

I did, and if the user is not an admin they never reach the code. I'm stil looking at moderators, it is not working as I had hoped.

Reply:
So it doesn't work as expected?

Reply:
It does, to clarify;

> I did, and if the user is not an admin they never reach the code

is intended behavior. 

I was experimenting with the code though and found out that under the 'right' circumstances var checks can raise exceptions in Twig while in fact trying to prevent them. I don't like that, So, despite it working as is I want it to, I've rewritten it and am currently testing a different strategy